main/windows/index.ts

```diff
@@ -554,35 +554,59 @@ const init = () => {
 
   store.observer(() => broadcast('permissions', JSON.stringify(store('permissions'))))
   store.observer(() => {
-    const summonShortcut = store('main.shortcuts.summon')
-    const accelerator = getAcceleratorFromShortcut(summonShortcut)
-    try {
-      globalShortcut.unregister(accelerator)
-      if (summonShortcutAccelerator) {
-        globalShortcut.unregister(summonShortcutAccelerator)
+    const keyboardLayout = store('keyboardLayout')
+    let summonShortcut: Shortcut = store('main.shortcuts.summon')
```

Comment:
this can be const if we're just re-assigning one of the fields

Reply:
Actually we're reassigning on L598

Reply:
yeah but do we need to?

Reply:
why was this marked as resolved?

Reply:
Oh ok so I made some changes and thought this was covered.  I had some issues with types, will check again when I get back